workshop/serializers.py

```diff
@@ -121,6 +129,64 @@ class Meta:
             'linkedin_url', 'youtube_url')
 
 
+class EntityDetailSerializer(serializers.ModelSerializer):
+    point_of_contact = serializers.SerializerMethodField()
+    is_subscribed = serializers.SerializerMethodField()
+    subscribed_users = serializers.SerializerMethodField()
+    is_por_holder = serializers.SerializerMethodField()
+
+    @swagger_serializer_method(serializer_or_field=UserProfileSerializer(many=True))
+    def get_point_of_contact(self, obj):
+        """
+        Joint Secretary of the Entity
+        """
+        serializer = UserProfileSerializer(obj.point_of_contact, many=True)
+        return serializer.data
+
+    @swagger_serializer_method(serializer_or_field=serializers.BooleanField)
+    def get_is_subscribed(self, obj):
+        """
+        true, if the user has subscribed to the entity, otherwise false
+        null, in case of anonymous login
+        """
+        user = self.context['request'].user
+        if not user.is_authenticated:
+            return None
+        # pylint: disable=no-member
+        profile = UserProfile.objects.get(user=user)
+        return profile in obj.subscribed_users.all()
+
+    @swagger_serializer_method(serializer_or_field=serializers.IntegerField)
+    def get_subscribed_users(self, obj):
+        """
+        Total number of users subscribed to the entity
+        """
+        return obj.subscribed_users.count()
+
+    @swagger_serializer_method(serializer_or_field=serializers.BooleanField)
+    def get_is_por_holder(self, obj):
+        """
+        true, if the user is the POR Holder of the Entity, otherwise false
+        """
+        user = self.context['request'].user
+        if not user.is_authenticated:
+            return False
+        # pylint: disable=no-member
+        profile = UserProfile.objects.get(user=user)
+        if obj in profile.get_club_privileges():
+            return True
+        return False
```

Comment:
```suggestion
    @swagger_serializer_method(serializer_or_field=serializers.BooleanField)
    def get_is_por_holder(self, obj):
        """
        true, if the user is the POR Holder of the Entity, otherwise false
        """
        user = self.context['request'].user
        if not user.is_authenticated:
            return False
        # pylint: disable=no-member
        profile = UserProfile.objects.get(user=user)
        if obj in profile.get_entity_privileges():
            return True
        return False
```